etna/whatson/blocks.py

```diff
@@ -1,7 +1,23 @@
 from wagtail import blocks
 
-from etna.core.blocks import ParagraphBlock
+from etna.core.blocks import PageListBlock, ParagraphBlock
 
 
 class EventPageBlock(blocks.StreamBlock):
     paragraph = ParagraphBlock()
+
+
+class RelatedArticlesBlock(blocks.StreamBlock):
```

Comment:
Question: is this for use on ExhibitionPages only? If so it might be helpful to name it more specifically.

Reply:
I think at some point we might add it to other pages - we tend to make things specific and then expand them to be generic, which makes it a bit more difficult to change down the line, so we've been trying to go a bit more "generic" with naming recently 😄 Good point though, I agree!